### PR TITLE
Stm32h7 update v1.9.1

### PR DESCRIPTION
**Pull request by ABOSTM**
**Summary**
Stm32h7 update v1.9.1

Validation
Nucleo_h743ZI2:
* I2C scanner + USB CDC (generic 'Serial' supersede U(S)ART)
* SD card info
* Analog read

### CODE REVIEW

File: system/STM32H7xx/system_stm32h7xx.c [hunk 1]
```diff
@@ -74,7 +73,49 @@
...
+#endif /* VECT_TAB_SRAM */
+#else
+/*!< Define VECT_TAB_SRAM if you need to relocate your vector Table
+
+     in Sram else user remap will be done in Flash. */
+#if defined(VECT_TAB_SRAM)
+#define VECT_TAB_BASE_ADDRESS   D1_AXISRAM_BASE   /*!< Vector Table base address field.
+                                                       This value must be a multiple of 0x200. */
+#define VECT_TAB_OFFSET         0x00000000U       /*!< Vector Table base offset field.
+                                                       This value must be a multiple of 0x200. */
```
Comment (fpistm) [inline system/STM32H7xx/system_stm32h7xx.c:109]: to be removed

File: system/STM32H7xx/system_stm32h7xx.c [hunk 2]
```diff
@@ -74,7 +73,49 @@
...
+#else
+#define VECT_TAB_BASE_ADDRESS   FLASH_BANK1_BASE  /*!< Vector Table base address field.
+                                                       This value must be a multiple of 0x200. */
+#define VECT_TAB_OFFSET         0x00000000U       /*!< Vector Table base offset field.
+                                                       This value must be a multiple of 0x200. */
```
Comment (fpistm) [inline system/STM32H7xx/system_stm32h7xx.c:114]: to be removed

**Review by fpistm — CHANGES_REQUESTED** [review]

File: system/STM32H7xx/system_stm32h7xx.c (hunk 1 above)
Comment (ABOSTM) [reply system/STM32H7xx/system_stm32h7xx.c]: oops sorry. Done

File: system/STM32H7xx/system_stm32h7xx.c (hunk 2 above)
Comment (ABOSTM) [reply system/STM32H7xx/system_stm32h7xx.c]: Done

**Review by fpistm — APPROVED** [review]

**Pull request merged**